content/programming_model_overview.tex

```diff
@@ -24,7 +24,7 @@
 PGAS languages, \openshmem relies on the user to use the library calls  to
 implement the correct semantics of its programming model.
 
-An overview of the \openshmem routines is described below:
+An overview of the \openshmem routines is described below. Remotely accessible data objects are referred to as \emph{Symmetric Data Objects} in this overview:
```

Comment:
Symmetric data objects should be italicized above when it is first introduced, not here.  Also, why is this term capitalized?

Reply:
If these edits were to make it into the spec, this would be the first time the term 'symmetric data objects' is introduced/mentioned.